### DISCUSSION TITLE
How to display mobilphone on People Serach result

### DISCUSSION THREAD

**Discussion by vizsnyai** [microsoft-search/pnp-modern-search#3394] [Q&A]
This solution is very good. Thanks
I have one issue.

The other task that I would like to know how it can be solved is the display of the MobilPhone value for results of the People type. I can't get this attribute to appear. I tried the slots 
{{slot item @root.slots.MobilPhone}};˙
{{slot item @root.slots.MobilePhone}};
{{slot item @root.slots.Mobile}}; 
{{slot item @root.slots.CellPhone}}; 
but no data came back. How could this be solved?

Thank you in advance for your help.

**Comment by till-llit** [comment]
the UPA Field is usally called MobilePhone. There is no predefined slot for this field.

make sure it is selected in the "Selected Properties" of the Search Results.

you can either address it direct: "{{item.MobilePhone}}" or create a new layout slot for this field.

Check the debug layout to see the raw data of the search results to verify if there is data in MobilePhone. If not -> you must check your SharePoint User Profile Application to see if the data is available in the profiles.
